packages/jsii-spec/lib/spec.ts

```diff
@@ -451,10 +451,32 @@ export interface Parameter extends Documentable {
     type: TypeReference;
 
     /**
-     * Whather this argument is the "rest" of a variadic signature.
-     * The ``#type`` is that of every individual argument of the variadic list.
+     * Denotes the modifier for this parameter, if any.
+     *
+     * @default no modifier
      */
-    variadic?: boolean;
+    modifier?: ParameterModifier;
+}
+
+/**
+ * Modifiers for a parameter.
+ */
+export enum ParameterModifier {
+    /**
+     * Indicates that the parameter may be omitted. When a parameter has the
+     * `Optional` midufuer, the following must also be true:
```

Comment:
fun typo :)

Reply:
Oh there be many midufüren around here ...